src/main/kotlin/no/skatteetaten/aurora/boober/service/GitService.kt

```diff
@@ -135,9 +135,24 @@ open class GitService(
         }
     }
 
-    fun commitAndPushChanges(repo: Git, ref: String = "refs/heads/master", commitMessage: String? = null) {
-
-        repo.add().addFilepattern(".").call()
+    /**
+     * When using addFilePattern or rmFilePattern there might be some changes that will not be committed.
+     * For removing unwanted changes use [cleanRepo].
+     */
+    fun commitAndPushChanges(
+        repo: Git,
+        ref: String = "refs/heads/master",
+        commitMessage: String? = null,
+        addFilePattern: String? = ".",
```

Comment:
Er det vits å ha med addFilePattern når den ikke brukes?

Reply:
Den brukes i VaultService.deleteVault hvor den settes til null for å forsikre oss at ikke alle endringer som er gjort i repoet blir tatt med.